primitives/src/crh/poseidon/parameters/bn382.rs

```diff
@@ -0,0 +1,484 @@
+use crate::crh::{
```

Comment:
Just for completeness: the parameters here are probably not reproducible, beside that the claimed generation script is not the updated one.

Reply:
Re-generated using yours